### PR TITLE
[#1035] Add FAQ section to documentation.

### PR DESCRIPTION
**Pull request by b-abel**
Add FAQ page and a couple of questions to start with.

Signed-off-by: Abel Buechner-Mihaljevic <Abel.Buechner@bosch-si.com>

### CODE REVIEW

**Comment by sophokles73** [comment]
@b-abel can you please rebase your PR so that the Travis build has a chance of succeeding?

**Comment by b-abel** [comment]
Done.

File: site/content/faq.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
++++
+title = "Frequently Asked Questions"
+menu = "main"
+menuTitle = "FAQs"
+weight = 550
+pre = '<i class="far fa-question-circle"></i> '
```
Comment (sophokles73) [inline site/content/faq.md]: what's this?

File: site/content/faq.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
++++
+
+
+## Technical Questions
+
+
+#### Why do I get `HTTP/1.1 503 Service Unavailable` when sending messages to the HTTP protocol adapter?
+
+Please check if you have a [consumer connected]({{< relref "getting-started.md#starting-a-consumer" >}}) 
+and that you do not filter out the message type.
```
Comment (sophokles73) [inline site/content/faq.md]: IMHO we should be more specific about the *message type* here, i.e. 
> and that your consumer is for the same type of message (*telemetry* or *event*) that you are sending.

File: site/content/faq.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Technical Questions
+
+
+#### Why do I get `HTTP/1.1 503 Service Unavailable` when sending messages to the HTTP protocol adapter?
+
+Please check if you have a [consumer connected]({{< relref "getting-started.md#starting-a-consumer" >}}) 
+and that you do not filter out the message type.  
+
+
+#### Why do I get the exception `io.vertx.core.VertxException: OpenSSL is not available`?
```
Comment (sophokles73) [inline site/content/faq.md]: ... during startup of protocol adapter X?

File: site/content/faq.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Please check if you have set the property `nativeTlsRequired` in the protocol adapter's configuration. The default Hono
```
Comment (sophokles73) [inline site/content/faq.md]: (set the property) ... to `true`

File: site/content/faq.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+containers do not contain `netty-tcnative`. To enable this option, please follow the explanation in the 
+[Admin Guide]({{< relref "admin-guide/secure_communication.md#using-openssl" >}}) or build your own images.
```
Comment (sophokles73) [inline site/content/faq.md]: ... container images.

File: site/content/faq.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+The configured maximum number of concurrent connections is exceeded and the protocol adapter refuses to accept more 
+connections to prevent running out of resources. This limit is either configured on the protocol adapter
+([MQTT]({{< relref "admin-guide/mqtt-adapter-config.md#service-configuration" >}}),
+[AMQP]({{< relref "admin-guide/amqp-adapter-config.md#service-configuration" >}})) or if not set, 
+the protocol adapter determines a reasonable value based on the available resources like memory and CPU.
+
+
+#### Why do I see `MemoryBasedConnectionLimitStrategy - Not enough memory` in the logs of a protocol adapter? 
+
+The protocol adapter can not allocate enough memory. Please provide more memory. To try it anyways, configure the
```
Comment (sophokles73) [inline site/content/faq.md]: cannot allocate enough memory for what?

File: site/content/faq.md (hunk 1 above)
Comment (b-abel) [reply site/content/faq.md]: It is an icon. This is not used with the current theme "material-docs", but will be shown in theme "learn". I can remove it here and add it after the merge of #1111.

**Pull request merged**